src/components/views/settings/CrossSigningPanel.js

```diff
@@ -74,12 +74,14 @@ export default class CrossSigningPanel extends React.PureComponent {
         const secretStorageKeyInAccount = await secretStorage.hasKey();
```

Comment:
Maybe remove the XXX comment above? It's probably okay for this debug panel to access private fields.